src/useObservable.js

```diff
@@ -0,0 +1,14 @@
+import { useEffect } from 'react';
+
+/**
+ * Hook, which helps you combine rxjs flow and setState in your component
+ */
+const useObservable = (observable, setter) => {
+  useEffect(() => {
+    const subscription = observable.subscribe(setter);
+
+    return () => subscription.unsubscribe();
```

Comment:
Perhaps this could be shortened by:

```
return subscription.unsubscribe;
```

Reply:
Yeah, I thought about it, but it works weird with `renderHook` in the test.
It failed with error:
```log
 "after each" hook for "should return a function":
 TypeError: Cannot read property 'closed' of null
```
🤔

Reply:
All right, never mind then ;)